src/components/SelectedAnnotation.jsx

```diff
@@ -52,12 +52,9 @@ class SelectedAnnotation extends React.Component {
 
   componentWillUnmount() {
     document.removeEventListener('keyup', this.handleKeyUp);
-    this.inputText.removeEventListener('mousedown', () => {
-      this.dialog.className = DISABLE_DRAG;
-    });
-    this.inputText.removeEventListener('mouseup', () => {
-      this.dialog.className = ENABLE_DRAG;
-    });
+    if (this.inputText) {
```

Comment:
I don't understand why the `if ()` existential check is _outside_ `removeEventListeners()`

Logically, I'd imagine it to be something like...

```
removeEventListeners() {
  if (this.inputText) {
    this.inputText.removeEventListener('mouseup', this.handleMouseUp);
    this.inputText.removeEventListener('mousedown', this.handleMouseDown);
  }
}
```

...which ensures that `removeEventListeners()` can be called willy-nilly and it'd still be safe to do so.